### PR TITLE
Fix some issues and improve codes

### PR DESCRIPTION
**Pull request by yoichiro**
The current Smart Home Washer codelab has unfortunately some issues.
https://codelabs.developers.google.com/codelabs/smarthome-washer/index.html#0

Actually, the issues are the following:

* Step3
  * There is no "Card containing the sign-in UI" elements. Therefore, an error occurs at refer the element from JavaScript code [here](https://github.com/googlecodelabs/smarthome-washer/blob/master/washer-start/public/main.js#L56).
* Step 4
  * `"pausable: true` is necessary to use the `RunCycle` trait and `isRunning` and `isPaused` values.
  * The undefined value `k` is used in the `onExecute` function. Therefore, an error occurs in runtime.
  * Both helper functions `queryDevice` and `queryFirebase` are missing in the washer-start. Therefore, an error occurs in the `onExecute` function at runtime.
* Step 8
  * There are unsupported properties in the parameter which are passed to the HomeGraph API. Therefore, the API returns the INVALID_PARAMETERS error.

This pull request has some commits to fix these issues.

### CODE REVIEW

**Comment by yoichiro** [comment]
This pull request consists of some commits to fix the following:

* Step3
  * [5699d0b] There is no "Card containing the sign-in UI" elements. Therefore, an error occurs at refer the element from JavaScript code here.
* Step 4
  * [1bd6440] "pausable: true is necessary to use the RunCycle trait and isRunning and isPaused values.
  * [a8c2c58] The undefined value k is used in the onExecute function. Therefore, an error occurs in runtime.
 * [33d335e] Both helper functions queryDevice and queryFirebase are missing in the washer-start. Therefore, an error occurs in the onExecute function at runtime.
* Step 8
  * [092f3f3] There are unsupported properties in the parameter which are passed to the HomeGraph API. Therefore, the API returns the INVALID_PARAMETERS error.
* Other
  * [574443e] For unifying the code format.

File: washer-start/functions/index.js [hunk 1]
```diff
@@ -69,16 +69,32 @@ const app = smarthome({
...
 
   };
 //  return {};
 });
 
+const queryFirebase = (deviceId) => firebaseRef.child(deviceId).once('value')
+  .then((snapshot) => {
+    const snapshotVal = snapshot.val();
+    return {
+      on: snapshotVal.OnOff.on,
```
Comment (Fleker) [inline washer-start/functions/index.js]: It's fine to return all of the parameters in `queryFirebase` as a response to a `QUERY` intent. The Report State call does need to be more restrictive.

File: washer-start/public/index.html [hunk 2]
```diff
@@ -45,6 +45,20 @@ <h3 class='user-welcome-heading' id='user-welcome'></h3>
     </header>
     <main class="mdl-layout__content mdl-color--grey-100">
       <div class="mdl-cell--16-col mdl-cell--16-col-tablet mdl-grid">
+
+        <!-- Card containing the sign-in UI -->
```
Comment (Fleker) [inline washer-start/public/index.html]: We can get rid of the signed-out-card since this codelab doesn't do any client-side auth.

**Comment by Fleker** [comment]
Thanks for making these changes.

File: washer-start/public/index.html (hunk 2 above)
Comment (yoichiro) [reply washer-start/public/index.html]: @Fleker Ok, I see. I thought that we have following two options to fix the issue. I understand that you want to apply the 2nd option below.

* Add the element set for the "Card containing the sign-in UI".
* Remove the line which refers the element at [washer-start/public/main.js#L56](https://github.com/googlecodelabs/smarthome-washer/blob/master/washer-start/public/main.js#L56).

I intend to delete the 5699d0b Then I will commit the code to achieve the 2nd option above.

File: washer-start/functions/index.js (hunk 1 above)
Comment (Fleker) [reply washer-start/functions/index.js]: I understand.

File: washer-start/public/index.html (hunk 2 above)
Comment (Fleker) [reply washer-start/public/index.html]: Yes, there's not much reason to have sign-in. Thanks.

**Comment by yoichiro** [comment]
@Fleker I delete the signed-out-card from `index.html` and `main.js` of both `washer-start` and `washer-done` folders at cb476b4. Could you check them?

**Comment by Fleker** [comment]
This looks good to me. Thanks again. I'm going to merge this in.

**Pull request merged**